cristin-funding/src/test/resources/success.json

```diff
@@ -0,0 +1,3972 @@
+[
+  {
+    "code": "EC/FP7",
```

Comment:
So this is where all the lines comes from. I thought I had a very big PR but 4/5 of lines is from this file. 

Do you need every entry in cristin? Does it load fast when testing?

Reply:
I don't need them all. I will reduce it.